test/test_interfaces.py

```diff
@@ -57,7 +71,95 @@ def test_examples_not_valid_path(self):
         with self.assertRaises(FileNotFoundError):
             interface = Interface(lambda x: x, "textbox", "label", examples='invalid-path')
             interface.launch()
+            
+    def test_test_launch(self):
```

Comment:
I feel like this is testing the internal implementation, not so much the desired behavior of `test_launch()`, which is just:
* `test_launch()` should *not* throw an error if the method/interface combination is valid
* `test_launch()` should throw an error if the method/interface combination is invalid